drivers/netdev_ieee802154/netdev_ieee802154.c

```diff
@@ -279,4 +279,21 @@ int netdev_ieee802154_dst_filter(netdev_ieee802154_t *dev, const uint8_t *mhr)
     return 1;
 }
 
+int netdev_ieee802154_data_frame_filter(netdev_ieee802154_t *dev,
+                                        const uint8_t *mhr, size_t frame_len)
+{
+    if (frame_len < IEEE802154_FCF_LEN) {
+        return 1;
```

Comment:
Shouldn't this be zero?
https://github.com/RIOT-OS/RIOT/pull/11191/commits/ae120c56f077066d49d69a6545e4902184a29995#diff-864e98442504008c9fad4791703b0e35R144